modules/clientBinaryManager.js

```diff
@@ -180,7 +180,9 @@ class Manager extends EventEmitter {
         .then((localConfig) => {
             if (!localConfig) {
                 log.info('No config for the ClientBinaryManager could be loaded, using local clientBinaries.json.');
-                localConfig = require('../clientBinaries.json');
+
+                const localConfigPath = path.join(Settings.userDataPath, 'clientBinaries.json');
```

Comment:
The statement in L#87 can only be reached if the computer is online. Otherwise it will simply return to the next promise-chain block. The skipped validation doesn't seem to be necessary as this did happen before as the `clientBinaries.json` was created.